components/brave_ads/core/internal/account/issuers/issuers.cc

```diff
@@ -92,13 +104,24 @@ void Issuers::FetchCallback(const mojom::UrlResponseInfo& url_response) {
   SuccessfullyFetchedIssuers(*issuers);
 }
 
+void Issuers::Reset() {
+  BLOG(1, "Reset issuers");
+
+  is_initialized_ = false;
+
+  CancelFetch();
+  return ResetIssuers();
+}
+
 void Issuers::SuccessfullyFetchedIssuers(const IssuersInfo& issuers) {
   StopRetrying();
 
   if (delegate_) {
     delegate_->OnDidFetchIssuers(issuers);
   }
 
+  is_initialized_ = true;
```

Comment:
maybe rename to somethink like was_successfully_fetched_

Reply:
Changed, thanks